custom_components/trakt_tv/apis/trakt.py

```diff
@@ -390,6 +390,59 @@ async def fetch_recommendations(self, configured_kinds: list[TraktKind]):
 
         return res
 
+    # start of new code
+    async def fetch_anticipated(self, path: str, limit: int):
+        return await self.request(
+            "get", f"{path}/anticipated?limit={limit}&ignore_collected=false"
```

Comment:
ignore_collected should probably be user's choice, i.e. set true or false from config